doc/source/examples/dem/small-scale-rotating-drum-post-processing/small-scale-rotating-drum-post-processing.rst

```diff
@@ -0,0 +1,216 @@
+  sys.path.append(path_to_module)
+  from lethe_pyvista_tools import *
+
+where `sys.path <https://docs.python.org/3/library/sys.html#sys.path:~:text=in%20version%203.10.-,sys.path%C2%B6,-A%20list%20of>`_ is a list of strings that specifies the search path for modules. However, one can simply copy the `lethe_pyvista_tools.py <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ file to the same folder as the Python post-processing script is and simply import it, such as:
+ 
+.. code-block::
+
+  from lethe_pyvista_tools import *
+
+The ``*`` means that we want to import all members of lethe_pyvista_tools. 
+
+Constructing the object
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following step is to create an object to receive the data. In the present case, the object is called ``particles``.
+
+.. code-block::
+
+  particles = lethe_pyvista_tools(case_path = ".", prm_file_name = "small-rotating-drum-dem.prm")
+
+Here, the constructor ``lethe_pyvista_tools`` receives two arguments: ``case_path`` and ``prm_file_name``. In the above code line, ``"."`` means that the ``case_path`` is the path where we currently are, but it can be any path where the case is. The ``prm_file_name`` argument takes ``"small-rotating-drum-dem.prm"``.
+
+.. tip::
+  Together with the object ``particles``, ``lethe_pyvista_tools`` creates a dictionary with all parameters in the ``.prm`` file. To access the parameter, we can use ``particles.prm_dict['$NAME_OF_THE_PARAMETER']``. In the present case for example, the diameter of the particles can be easily printed using ``print(particles.prm_dict['diameter'])``. This can be useful for post-processing routines with multiple simulations.
+
+Reading Lethe results to Python
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next, we read the results into Python:
+
+.. code-block::
+  
+  particles.read_lethe_to_pyvista(pvd_name = "out.pvd")
+
+``read_lethe_to_pyvista`` takes the name of the ``.pvd`` file generated by the simulation. The method can take 3 other arguments: ``first``, ``last``, and ``interval``, standing for the first and last time-steps to be read and the interval between the time-steps, respectively. By default, ``first = 0``, ``interval = 1``, and ``last`` is the last time-step of the data.
+
+The ``read_lethe_to_pyvista`` reading function assigns the datasets of each time-step to the object ``particles``. Each time-step corresponds to a `PyVista dataset <https://docs.pyvista.org/user-guide/vtk_to_pyvista.html#>`_, and can be accessed using ``particles.df[$TIME-step_NUMBER]``.
+
+.. note:: 
+  
+  If we set ``first = 2``, ``particles.df[0]`` corresponds to the dataset of the 2nd time-step of the simulation.
+
+
+Creation of a new array
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To color the particles according to their position, we use the function ``array_modifier``, which takes the following arguments:
+
+- ``reference_array_name``: Name of the array used to sort the data. By default: ``"ID"``
```

Comment:
I know that saying "array used to sort the data" and "ID" means you use the ID of particle for sorting, but what else can it be? If there's other nice option, how can it be relevant?